subsys/net/l2/canbus/canbus.c

```diff
@@ -186,6 +220,13 @@ static inline void cpy_lladdr(struct net_pkt *dst, struct net_pkt *src)
 	lladdr->len = net_pkt_lladdr_dst(src)->len;
 	lladdr->type = NET_LINK_CANBUS;
 
+#ifdef CONFIG_NET_L2_CANBUS_ETH_TRANSLATOR
+	if (dest_is_translator(src)) {
+		/*Make room for address extension*/
```

Comment:
Please add spaces here `/* Make room .... extension */`
Can you check all the comments for these and fix them all.